pkg/dyninst/testprogs/probes.go

```diff
@@ -52,3 +52,47 @@ func GetProbeCfgs(t *testing.T, name string) []config.Probe {
 	}
 	return probesCfgs
 }
+
+// GetExpectedOutput returns the expected output for a given service.
+func GetExpectedOutput(t *testing.T, name string) map[string]string {
```

Comment:
I do not find it very reasonable to have this function or the output of a test in another package to be underneath `testprogs`. This should be *much* closer to the test that uses it and the testdata for that test should be underneath its own package.